[INFO] [2021-06-15 22:05:17,351] [4034:140389192021824] - control_client.py[line:42]: request update job 2021061522050718069622 task 2021061522050718069622_dataio_0 0 on guest 9999
[DEBUG] [2021-06-15 22:05:17,787] [4034:140389192021824] - _federation.py[line:15]: [federation.standalone]init federation: standalone_session=<fate_arch._standalone.Session object at 0x7faee5d8da20>, federation_session_id=2021061522050718069622_dataio_0_0, party=Party(role=guest, party_id=9999)
[DEBUG] [2021-06-15 22:05:17,795] [4034:140389192021824] - _federation.py[line:23]: [federation.standalone]init federation context done
[INFO] [2021-06-15 22:05:17,797] [4034:140389192021824] - tracker_client.py[line:150]: Request read job 2021061522050718069622 task None None on guest 9999 data data info
[INFO] [2021-06-15 22:05:17,825] [4034:140389192021824] - task_executor.py[line:269]: load computing table use 4
[DEBUG] [2021-06-15 22:05:17,891] [4034:140389192021824] - component_properties.py[line:73]: {'DataIOParam': {'input_format': 'dense', 'delimitor': ',', 'data_type': 'float64', 'exclusive_data_type': None, 'tag_with_value': False, 'tag_value_delimitor': ':', 'missing_fill': False, 'default_value': 0, 'missing_fill_method': None, 'missing_impute': None, 'outlier_replace': False, 'outlier_replace_method': None, 'outlier_impute': None, 'outlier_replace_value': 0, 'with_label': True, 'label_name': 'y', 'label_type': 'int', 'output_format': 'dense', 'need_run': True}, 'initiator': {'role': 'guest', 'party_id': 9999}, 'component_parameters': {'role': {'host': {'0': {'reader_0': {'table': {'name': 'breast_hetero_host', 'namespace': 'experiment'}}, 'reader_1': {'table': {'name': 'breast_hetero_host', 'namespace': 'experiment'}}, 'dataio_1': {'with_label': False}, 'dataio_0': {'with_label': False}}}, 'guest': {'0': {'reader_0': {'table': {'name': 'breast_hetero_guest', 'namespace': 'experiment'}}, 'reader_1': {'table': {'name': 'breast_hetero_guest', 'namespace': 'experiment'}}, 'dataio_1': {'with_label': True, 'output_format': 'dense'}, 'dataio_0': {'with_label': True, 'output_format': 'dense'}}}}, 'common': {'hetero_secure_boost_0': {'task_type': 'classification', 'objective_param': {'objective': 'cross_entropy'}, 'num_trees': 3, 'validation_freqs': 1, 'encrypt_param': {'method': 'iterativeAffine'}, 'tree_param': {'max_depth': 3}}, 'evaluation_0': {'eval_type': 'binary'}}}, 'job_parameters': {'job_type': 'predict', 'work_mode': 0, 'backend': 0, 'computing_engine': 'STANDALONE', 'federation_engine': 'STANDALONE', 'storage_engine': 'STANDALONE', 'engines_address': {'computing': {'nodes': 1, 'cores_per_node': 20}, 'federation': {'nodes': 1, 'cores_per_node': 20}, 'storage': {'nodes': 1, 'cores_per_node': 20}}, 'federated_mode': 'SINGLE', 'task_parallelism': 1, 'computing_partitions': 4, 'federated_status_collect_type': 'PULL', 'model_id': 'guest-9999#host-9998#model', 'model_version': '2021061522023585229020', 'eggroll_run': {'eggroll.session.processors.per.node': 4}, 'spark_run': {}, 'rabbitmq_run': {}, 'pulsar_run': {}, 'adaptation_parameters': {'task_memory_per_node': 0, 'if_initiator_baseline': False, 'task_cores_per_node': 4, 'request_task_cores': 4, 'task_nodes': 1}}, 'dsl_path': './examples/LDPDev/dsl/v2/hetero_secureboost/test_secureboost_train_dsl.json', 'conf_path': './examples/LDPDev/test_predict_conf.json', 'dsl_version': 2, 'role': {'host': [9998], 'guest': [9999]}, 'local': {'role': 'guest', 'party_id': 9999}, 'CodePath': 'federatedml/util/data_io.py/DataIO', 'module': 'DataIO', 'output_data_name': ['data']}
[DEBUG] [2021-06-15 22:05:17,891] [4034:140389192021824] - component_properties.py[line:80]: need_run: True, need_cv: False
[WARNING] [2021-06-15 22:05:17,891] [4034:140389192021824] - data_io.py[line:855]: DataIO is deprecated, and will be removed in 1.7, use DataTransform module instead
[DEBUG] [2021-06-15 22:05:17,892] [4034:140389192021824] - component_properties.py[line:102]: parse_dsl_args data_sets: {'reader_0': {'data': [<fate_arch.computing.standalone._table.Table object at 0x7faea2cb8828>]}}
[DEBUG] [2021-06-15 22:05:17,892] [4034:140389192021824] - component_properties.py[line:112]: [Data Parser], has_train_data: False
[DEBUG] [2021-06-15 22:05:17,892] [4034:140389192021824] - component_properties.py[line:112]: [Data Parser], has_eval_data: False
[DEBUG] [2021-06-15 22:05:17,892] [4034:140389192021824] - component_properties.py[line:112]: [Data Parser], has_validate_data: False
[DEBUG] [2021-06-15 22:05:17,892] [4034:140389192021824] - component_properties.py[line:112]: [Data Parser], has_test_data: False
[DEBUG] [2021-06-15 22:05:17,892] [4034:140389192021824] - component_properties.py[line:118]: [Data Parser], has_normal_data: True
[DEBUG] [2021-06-15 22:05:17,893] [4034:140389192021824] - component_properties.py[line:179]: Input data_sets: {'reader_0': {'data': [<fate_arch.computing.standalone._table.Table object at 0x7faea2cb8828>]}}
[DEBUG] [2021-06-15 22:05:17,893] [4034:140389192021824] - component_properties.py[line:189]: data_dict: {'data': [<fate_arch.computing.standalone._table.Table object at 0x7faea2cb8828>]}
[DEBUG] [2021-06-15 22:05:17,893] [4034:140389192021824] - component_properties.py[line:192]: data_list: <fate_arch.computing.standalone._table.Table object at 0x7faea2cb8828>
[DEBUG] [2021-06-15 22:05:17,902] [4034:140389192021824] - component_properties.py[line:231]: train_data: None, validate_data: None, test_data: None, data: {'reader_0.data': <fate_arch.computing.standalone._table.Table object at 0x7faea2cb8828>}
[DEBUG] [2021-06-15 22:05:17,902] [4034:140389192021824] - model_base.py[line:88]: running_funcs: [<bound method DataIO.load_model of <federatedml.util.data_io.DataIO object at 0x7faea2ce55f8>>, <function ModelBase.extract_data at 0x7faea2ca6400>, <bound method ModelBase.set_flowid of <federatedml.util.data_io.DataIO object at 0x7faea2ce55f8>>, <bound method DataIO.transform of <federatedml.util.data_io.DataIO object at 0x7faea2ce55f8>>]
[DEBUG] [2021-06-15 22:05:17,902] [4034:140389192021824] - model_base.py[line:340]: In extract_data, data input: {'reader_0.data': <fate_arch.computing.standalone._table.Table object at 0x7faea2cb8828>}
[DEBUG] [2021-06-15 22:05:17,903] [4034:140389192021824] - model_base.py[line:97]: func: <bound method DataIO.transform of <federatedml.util.data_io.DataIO object at 0x7faea2ce55f8>>
[INFO] [2021-06-15 22:05:17,903] [4034:140389192021824] - data_io.py[line:118]: start to read dense data and change data to instance
[DEBUG] [2021-06-15 22:05:17,907] [4034:140389192021824] - data_io.py[line:75]: header is y,x0,x1,x2,x3,x4,x5,x6,x7,x8,x9
[DEBUG] [2021-06-15 22:05:17,907] [4034:140389192021824] - data_io.py[line:76]: sid_name is id
[DEBUG] [2021-06-15 22:05:18,015] [4034:140389192021824] - abnormal_detection.py[line:81]: schema is {'header': ['x0', 'x1', 'x2', 'x3', 'x4', 'x5', 'x6', 'x7', 'x8', 'x9'], 'sid_name': 'id', 'label_name': 'y'}
[DEBUG] [2021-06-15 22:05:18,016] [4034:140389192021824] - abnormal_detection.py[line:85]: header is ['x0', 'x1', 'x2', 'x3', 'x4', 'x5', 'x6', 'x7', 'x8', 'x9']
[DEBUG] [2021-06-15 22:05:18,016] [4034:140389192021824] - abnormal_detection.py[line:95]: sid_name is id
[DEBUG] [2021-06-15 22:05:18,016] [4034:140389192021824] - abnormal_detection.py[line:100]: label_name is y
[DEBUG] [2021-06-15 22:05:18,073] [4034:140389192021824] - io_check.py[line:35]: num row of input: 569 -> num row of output: 569
[DEBUG] [2021-06-15 22:05:18,122] [4034:140389192021824] - model_base.py[line:109]: saved_result is : [<fate_arch.computing.standalone._table.Table object at 0x7faee21c7908>], data_output: <fate_arch.computing.standalone._table.Table object at 0x7faee21c7908>
[INFO] [2021-06-15 22:05:18,122] [4034:140389192021824] - tracker_client.py[line:173]: Request save job 2021061522050718069622 task 2021061522050718069622_dataio_0 0 on guest 9999 component summary
[INFO] [2021-06-15 22:05:18,487] [4034:140389192021824] - control_client.py[line:42]: request update job 2021061522050718069622 task 2021061522050718069622_dataio_0 0 on guest 9999
[INFO] [2021-06-15 22:05:18,625] [4034:140389192021824] - control_client.py[line:42]: request update job 2021061522050718069622 task 2021061522050718069622_dataio_0 0 on guest 9999
